converter/plugin.go

```diff
@@ -0,0 +1,120 @@
+package converter
+
+import (
+	"net/rpc"
+
+	"github.com/hashicorp/go-plugin"
+	commonpb "go.temporal.io/api/common/v1"
+)
+
+type DataConverterRPC struct {
+	client *rpc.Client
+}
```

Comment:
Why is this in the SDK code? I thought we only plan to use this for the `tctl`.
I don't think we want SDK to depend on `go-plugin` pacakge.

Reply:
It can move to the `temporal` package if we're sure we won't want it for other purposes.